aztk/client/base/helpers/list_tasks.py

```diff
@@ -0,0 +1,27 @@
+from aztk.models import SchedulingTarget, Task
+
+from .get_recent_job import \
+    get_recent_job  # TODO: move this function to azkt.client
```

Comment:
todo?

Reply:
This was actually already done, just forgot to remove the TODO